package.json

```diff
@@ -12,7 +12,7 @@
     "test": "karma start"
   },
   "devDependencies": {
-    "@folio/stripes-core": "thefrontside/stripes-core#master",
+    "@folio/stripes-core": "2.7.0",
```

Comment:
We'll need #master until they release the changes to the redux store setup

Reply:
@wwilsman is that this PR? https://github.com/folio-org/stripes-core/pull/38 - if so, I'll change to the commit hash on `folio-org/stripes-core`; I'm very hesitant to point to `folio-org/stripes-core/master`.

Reply:
That's the one. :+1:

Reply:
Why not use the release version, or `#master` on our fork ?

Reply:
Yup, going to switch back to #master on our fork.